### PR TITLE
Fixing ESLint config and folder structure

### PR DESCRIPTION
**Pull request by Subham-Panda**
Fixing ESLint config, location of server.js, main script and description in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,24 +1,20 @@
 {
   "name": "codegolf-backend",
   "version": "1.0.0",
-  "description": "[![csivit][csivitu-shield]][csivitu-url] <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section --> [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-) <!-- ALL-CONTRIBUTORS-BADGE:END --> [![Issues][issues-shield]][issues-url]",
-  "main": "index.js",
+  "description": "Backend for a CodeBAsh Platform",
+  "main": "server.js",
```
Comment (roerohan) [inline package.json:5]: Change this to
```
src/server.js
```

File: .eslintrc.json [hunk 2]
```diff
@@ -1,21 +1,15 @@
...
+        "es2021": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
```
Comment (roerohan) [inline .eslintrc.json:13]: By default, the indent rule is 2 spaces. We prefer tabs tbh XD. You can add
```json
"rules": {
    "indent": [
        "error",
         4
    ]
}
```

**Review by roerohan — COMMENTED** [review]
Some small changes... @Subham-Panda

File: package.json (hunk 1 above)
Comment (Subham-Panda) [reply package.json]: required changes done

File: .eslintrc.json (hunk 2 above)
Comment (Subham-Panda) [reply .eslintrc.json:13]: rules added, indentation rule set

**Comment by roerohan** [comment]
Good job @Subham-Panda

**Pull request merged**